### PR TITLE
Update environmentVariables in sample job to make the structure clear.

### PR DESCRIPTION
**Pull request by ssk2**
Related to my pain in #389, it would have helped if the README was clearer. Here's a small change that should make things more obvious to fellow newbies!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -420,7 +420,8 @@ you can also use a url in the command field, if your mesos was compiled with cUR
    ],
    "schedule":"R/2014-03-08T20:00:00.000Z/PT2H",
    "environmentVariables": [
-     {"name": "FOO", "value": "BAR"}
+     {"name": "FOO", "value": "BAR"},
```
Comment (brndnmtthws) [inline README.md]: Maybe we should use more likely examples? like `JVMOPTS="-Xmx1000m"` and `JAVA_LIBRARY_PATH=/usr/local/lib`?

Comment (ssk2) [inline README.md]: Good suggestion!

**Comment by ssk2** [comment]
Updated @brndnmtthws.

**Comment by brndnmtthws** [comment]
:+1: 

Merge away.

**Pull request merged**